### DISCUSSION TITLE
Does the down arrow icon mean refresh?

### DISCUSSION THREAD

**Discussion by liamauve** [Kunzisoft/KeePassDX#1706] [General]
if I click it there's a pop up progress bar but's it's so fast im not sure what it says. probably "retrieving database..." I don't see anything changed afterwards, what's does it do? there's another icon with two arrows merging downward, but if I click it, it doesnt ask me what to merge with, again only a fast pop up progress bar that i'm not sure what it's doing. 
sorry im really new with keepass, please advise

**Comment by J-Jamet** [comment]
https://github.com/Kunzisoft/KeePassDX/wiki/File-Manager-and-Sync
Arrow down is "Reload data"
Two arrows is "Merge data"
If the URI providing the file has no new data than that loaded locally, you won't see any change.

**Reply by gh2013a** [reply]
New here. I've uninstalled KeePass using the Wise Uninstaller but still the annoying icons with the red X appear on every login screen. How to get rid of them?

**Reply by J-Jamet** [reply]
I don't know, your behavior is not related to KeePassDX.
